controllers/feeds/getFeedsV2.js

```diff
@@ -315,7 +311,7 @@ module.exports = async (req, res) => {
       if (data[i].own_reactions) {
         data[i].own_reactions = addKarmaScoreToLatestReaction(data[i].own_reactions, karmaScores);
       }
-
+      data[i] = modifyReactionsPost(data[i], data[i].anonimity);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `modifyReactionsPost` function is invoked here to alter the post object. It's important to ensure that this function doesn't mutate the original data unintentionally, as it could lead to unexpected side effects elsewhere in the code. If possible, consider making this function pure by having it return a new modified object instead of altering the original one.
```diff
- data[i] = modifyReactionsPost(data[i], data[i].anonimity);
+ data[i] = {...data[i], ...modifyReactionsPost(data[i], data[i].anonimity)};
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->